parse-json/README.md

```diff
@@ -183,7 +183,7 @@ fn get() -> impl Future<Item = (), Error = ()> {
 
 Two new things can be seen here.
 
-We used the `Vec` type here, because we get multiple `Repository`'s from the response. ([Remember](../read-files/README.md) that we already used the `vec!` macro which created a `Vec`.)
+We used the `Vec` type here, because we get multiple `Repository`'s from the response. ([Remember](../read-files/README.md) that we already used the `vec!` macro in previos chapter, which created a `Vec`.)
```

Comment:
```suggestion
We used the `Vec` type here, because we get multiple `Repository`'s from the response. ([Remember](../read-files/README.md) that we already used the `vec!` macro in a previous chapter, which created a `Vec`.)
```